### ISSUE TITLE
Folding switch color: node background, instead of node border

### ISSUE THREAD

**Issue by euu2021** [freeplane/freeplane#1453]
### Discussed in https://github.com/freeplane/freeplane/discussions/1452

<div type='discussions-op-text'>

<sup>Originally posted by **euu2021** October  7, 2023</sup>
First, some definitions:
Apparently, there are two different elements in FP (as per `Preferences…->Appearance->Size limits->Folding mark width ` and `Preferences…->Appearance->Size limits->Folding switch width`) :
- Folding mark: the mark that shows that a node has children
- Folding switch: the "button" that, once pressed, fold/unfold the node

---
My problem is that I use a dark background and dark node borders, so the Folding switch gets almost invisible:
![javaw_RsX1YHk3sN](https://github.com/freeplane/freeplane/assets/77707706/84a14c9d-d391-40c9-8b54-788b25a3dbfd)

In other words: 
- Folding mark is fine, because it follows the background color of the node
- Folding switch is not fine, because it follows the node border, and gets almost invisible

Having a visible Folding switch is useful, to know if the node has children.

I tried using lighter colors for the node borders, but I really prefer black.

Feature request Issue: 

Thanks,

</div>

**Comment by dpolivaev** [comment]
Please upload an example map. Getting your ideas from screenshots is too hard for me.

**Comment by euu2021** [comment]
The colors are like this:
[EXAMPLE colors.zip](https://github.com/freeplane/freeplane/files/12849773/EXAMPLE.colors.zip)


And I use a spotlight like this:
![image](https://github.com/freeplane/freeplane/assets/77707706/705fae56-513f-481b-a37a-4319ebf2eff4)

**Comment by AlexandreBonneau** [comment]
If you use colors close to each other, then you are bound to have difficulties seeing them.
It sounds to me that's a preference problem, not a Freeplane problem.

I also use exclusively dark mindmaps, and have not encountered this problem with the default line colors:
<img width="65" alt="2023-12-12_105203_Screenshot" src="https://github.com/freeplane/freeplane/assets/7248877/ef48ef35-4135-4e6f-9468-204c0b5faff3"> -> <img width="65" alt="2023-12-12_105045_Screenshot" src="https://github.com/freeplane/freeplane/assets/7248877/26d7a914-f225-4a02-87e5-7b153d567eb9">

The fact that besides the color which is bright, the switch also change size on hover is very clear to show the call to action button.
Anyway I do not want to discard your need (to each his own), so that's only my opinion :)

**Comment by euu2021** [comment]
> If you use colors close to each other, then you are bound to have difficulties seeing them.
> It sounds to me that's a preference problem, not a Freeplane problem.

For my map, the dark border works much better than a light border. See comparison:

![image](https://github.com/freeplane/freeplane/assets/77707706/a525c2ee-a56d-4c2b-8929-b90668e4c086)

As you can see, it's much easier to tell the nodes apart when the border is dark.

And I don't like to use colors for the boders, because I use color coding in the map, so adding colors for the borders would result in a complete visual mess. My borders need to be in shades of grey.

**Comment by EdoFro** [comment]
> * Folding switch is not fine, because it follows the node border, and gets almost invisible

I think the "folding switch" is different when the node is folded than when the node is unfolded.
unforlded:
![image](https://github.com/freeplane/freeplane/assets/35700575/c28474c7-59f6-478c-8b52-a2abccbe3971)

folded:
![image](https://github.com/freeplane/freeplane/assets/35700575/3e309215-9517-4fe7-b3c5-ed72c18ae2ef)

I think what you want is that its format is in both cases like the "unfolded" one. not?

Maybe it was made this way, so it looks different to the "Folding Mark" (if the user choose similar widths)

**Comment by EdoFro** [comment]
an idea:

### **unfolded "Folding Switch"** (same as today):
- border color = node's border color
- fill color = node's background color




### **folded "Folding Switch"** (change):
- border color = node's background color
- fill color = node's border color

**Comment by AlexandreBonneau** [comment]
I personally think the current version works great and has been so for many years, so if you were to change it, please make it an option :)
